index.js

```diff
@@ -2,7 +2,7 @@
 var fs = require('fs')
 var once = require('once')
 var split = require('split')
-var through = require('@ljharb/through')
+var { Writable } = require('node:stream')
```

Comment:
use of destructuring, or `node:`, would be a breaking change
```suggestion
var Writable = require('stream').Writable
```

also, Writable wasn't added until node 0.10, which technically would also be a breaking change and require an engines.node declaration to be added.

Reply:
chalk 4 requires node 10+, so older version aren't supported already.
and CI for hostile is only running on Node 14, so there was no indication for old version support to begin with.

Reply:
indeed you're right; https://github.com/feross/hostile/commit/6d5d1c37b627a553306a14991cb56e1dde125c96 was a breaking change in a patch version.

Again, lack of CI coverage indicates nothing; `engines.node` combined with "what actually worked at one point within the major" is the indicator. In other words, chalk needs to be dropped to 3 or replaced, *and* CI coverage should be expanded.

Reply:
@ljharb literally nobody complained over all this time, so looks like it's not a problem for anyone among real users of the library, so why fix something that isn't broken?

Reply:
being semver-compliant requires adhering to principle, regardless of whether there's actual/reported breakage or not.

It's up to the maintainer, of course, whether they want to comply with semver or not - I'm just helpfully pointing out what that requires.

Reply:
@feross what is your stance on this? if requested, I can do the obsolete Node compatibility changes in this PR, but it seems more reasonable for me to stay on Node 10+